src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -57,6 +57,7 @@ public String map() {
     public String saveCache(Cache cache){
         try {
             cacheService.save(cache);
+            logger.info(String.format("Your caches have been saved."));
```

Comment:
added info logger to notify upon caches saved

Reply:
Logging is a great idea.